### ISSUE TITLE
investigate why app-test jobs are sometimes hanging in CI

### ISSUE THREAD

**Issue by redshiftzero** [freedomofpress/securedrop#4691]
## Description

I'm seeing a lot of our [python app-test CI jobs hanging](https://circleci.com/gh/freedomofpress/securedrop/32430
), e.g.:

```
tests/functional/test_admin_interface.py::TestAdminInterface::test_admin_interface PASSED [  0%]
tests/functional/test_admin_interface.py::TestAdminInterface::test_admin_edits_hotp_secret PASSED [  0%]
tests/functional/test_admin_interface.py::TestAdminInterface::test_admin_edits_totp_secret make: *** [test] Terminated
Too long with no output (exceeded 10m0s)
```

We should investigate this - a re-run resolves but it's still annoying for devs/contributors

**Comment by emkll** [comment]
Another one: https://circleci.com/gh/freedomofpress/securedrop/33450

**Comment by eloquence** [comment]
Another one:
https://circleci.com/gh/freedomofpress/securedrop/35018

**Comment by eloquence** [comment]
Bump to 15 minutes does not appear to have materially impacted this flake; we still see failures at the HOTP/TOTP stage regularly, e.g.:

https://circleci.com/gh/freedomofpress/securedrop/36412

**Comment by eloquence** [comment]
I'm not able to spot this app test failure in recent runs, perhaps resolved via #4995?

**Comment by eloquence** [comment]
Closing as we don't seem to be seeing these failures anymore in recent builds.
